app/adapters/threema_adapter/outbound.rb

```diff
@@ -4,15 +4,26 @@ module ThreemaAdapter
   class Outbound < ApplicationJob
     queue_as :default
 
-    def self.send!(message)
-      perform_later(message)
+    def self.threema_instance
+      @threema_instance ||= Threema.new
     end
 
-    def perform(message)
+    def self.send!(message)
       recipient = message.recipient
-      return unless recipient&.threema_id
+      return unless message.recipient&.threema_id
+
+      perform_later(recipient: recipient, text: message.text)
+    end
+
+    def self.send_welcome_message!(contributor)
+      return unless contributor&.threema_id
+
+      welcome_message = [Setting.onboarding_success_heading, Setting.onboarding_success_text].join("\n")
```

Comment:
Threema also supports basic text formatting in messages, so maybe we could keep that consistent across channels and use bold text for the heading (as we do for Telegram)?

https://threema.ch/en/faq/markup